apps/react/src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,31 @@
+import { CircularProgress } from '@mui/material';
+import { FC, PropsWithChildren, memo } from 'react';
+
+import styles from './Loader.module.css';
+
+/** Loader props. */
+type LoaderProps = PropsWithChildren<{
+
+	/** Is loading. */
+	readonly isLoading: boolean;
+}>;
+
+/** Loader component. */
+const LoaderComponent: FC<LoaderProps> = ({ isLoading, children }) => {
```

Comment:
```suggestion
const LoaderComponent = ({ isLoading, children } as LoaderProps) => {
```

Don't use react `FC`. This approach has a number of cons. You can familiarize yourself with them in this [article](https://www.totaltypescript.com/you-can-stop-hating-react-fc).

Reply:
I have an error 
<img width="653" alt="image" src="https://github.com/saritasa-nest/camp-js-2023-roman-simernin/assets/75197645/dd331766-45bc-4a11-9c73-b2bc7e29fac6">

Reply:
So it works fine
<img width="420" alt="image" src="https://github.com/saritasa-nest/camp-js-2023-roman-simernin/assets/75197645/65ecd6ba-01dd-4fba-b57a-f75db915ad6d">